src/series/graph.serie.line.js

```diff
@@ -1703,10 +1702,10 @@ class SerieLine extends Serie {
 
     switch ( this.getStyle( selectionType ).lineStyle ) {
 
-      case 2:
+      case 1:
```

Comment:
> I rest my case, setLineStyle(1); should be the same as setLineStyle(0) and setLineStyle()

Then what should be here ? If it should be 2, what should be on the next case ? If next case should be 3, what should be in the next next case?
I repeat, there is a duplicate case, so the last one (currently the second case 3) is never reached.